docs/guides/getting-started-java-spring.md

```diff
@@ -0,0 +1,249 @@
+zeebe.client.broker.restAddress=http://127.0.0.1:8080
+zeebe.client.security.plaintext=true
+```
+
+### Create a worker
+
+1. In `src/main/java/io/camunda/demo/process_payments/`, create a file called `ChargeCreditCardWorker.java`.
+2. In the file created above, paste the following dependencies and package `package io.camunda.demo.process_payments`:
+
+```java
+package io.camunda.demo.process_payments;
+
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import io.camunda.zeebe.spring.client.annotation.JobWorker;
+import io.camunda.zeebe.spring.client.annotation.Variable;
+```
+
+3. Next, we can decorate our class with `@Component` and instantiate a logger. Additionally, we will add a method and decorate the `chargeCreditCard` method with `@JobWorker`, and specify a `@Variable(name = "totalWithTax") Double totalWithTax` argument into the `chargeCreditCard` method we'll implement:
+
+```java
+@Component
+public class ChargeCreditCardWorker {
+
+// Instantiate a logger
+  private final static Logger LOG = LoggerFactory.getLogger(ChargeCreditCardWorker.class);
+
+// Add the following method, and decorate `chargeCreditCard` with `@JobWorker`
+  @JobWorker(type = "charge-credit-card")
+
+// Specify a `@Variable(name = "totalWithTax") Double totalWithTax` argument into the `chargeCreditCard` method
+  public Map<String, Double> chargeCreditCard(@Variable(name = "totalWithTax") Double totalWithTax) {
+
+   // Implement the method
+    LOG.info("charging credit card: {}", totalWithTax);
+
+    return Map.of("amountCharged", totalWithTax);
+  }
+}
+```
+
+:::note
+To check your work, visit our [sample repository](https://github.com/camunda/camunda-8-get-started-spring/blob/main/src/main/java/io/camunda/demo/process_payments/ChargeCreditCardWorker.java) with the completed code.
+:::
+
+In your terminal, run `mvn spring-boot:run`, where you should see the `charging credit card` output. In Operate, refresh if needed, and note the payment has executed.
+
+## Step 7: Start a process instance
+
+To start a process instance programmatically, take the following steps:
+
+1. In `ProcessPaymentsApplication.java`, instantiate a static logger and declare `zeebeClient` with the `@Autowired` annotation. To convert the application to a `CommandLineRunner`, add `implements CommandLineRunner` to the class declaration `public class ProcessPaymentsApplication implements CommandLineRunner`.
+
+```java
+@SpringBootApplication
+@Deployment(resources = "classpath:process-payments.bpmn")
+public class ProcessPaymentsApplication implements CommandLineRunner {
+
+	private static final Logger LOG = LoggerFactory.getLogger(ProcessPaymentsApplication.class);
+
+	@Autowired
+	private ZeebeClient zeebeClient;
+
+	public static void main(String[] args) {
+		SpringApplication.run(ProcessPaymentsApplication.class, args);
+	}
+}
+```
+
+2. Implement an overriding `run` method:
```

Comment:
```suggestion
2. Implement an overriding `run` method in `ProcessPaymentsApplication`. When the application runs, it will create a new `process-payments` process instance, of the latest version, with specified variables, and send it to our local Self-Managed instance:
```

1. Especially because we are writing this class in two snippets, I think it's important to remind them where they're adding this method.
2. non-blocking: adding more description about what they're doing & why.

Reply:
Adjusted in latest commit 👍